cmd/oras/repository/show-tags.go

```diff
@@ -41,6 +44,12 @@ func showTagsCmd() *cobra.Command {
 Example - Show tags of the target repository:
   oras repository show-tags localhost:5000/hello
 
+Example - Show tags in the target repository with digest-like tags hidden:
+oras repository show-tags localhost:5000/hello --exclude-digest-tag
```

Comment:
Considering https://github.com/oras-project/oras/issues/667, can we replace `show-tags` with `tags` in all examples?

Reply:
Sure thing. It will be done in another PR.